06_puppy/runnerdave/types.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+// Puppy represents an item in the puppy store
+type Puppy struct {
+	Breed  string  `json:"name"`
+	Colour string  `json:"color"`
+	ID     uint16  `json:"id"`
+	Value  float32 `json:"value"`
```

Comment:
Not a big deal, but json is not required for this lab IMO.

Reply:
you are right but it will be used later